packages/varlet-ui/src/grid/props.ts

```diff
@@ -0,0 +1,52 @@
+import { toNumber } from '@varlet/shared'
+import { PropType } from 'vue'
+
+export const props = {
+  column: {
+    type: [String, Number],
+    default: 4,
+    validator(value: string | number) {
+      const val = toNumber(value)
+      return val > 0 && val <= 12
+    },
+  },
+  iconSize: {
+    type: [String, Number],
+    default: 24,
+  },
+  square: {
+    type: Boolean,
+    default: false,
+  },
+  gutter: {
+    type: [String, Number],
+    default: 0,
+  },
+  center: {
+    type: Boolean,
+    default: true,
+  },
+  border: {
+    type: Boolean,
+    default: true,
+  },
+  ripple: {
+    type: Boolean,
+    default: false,
+  },
+  elevation: {
+    type: [String, Number],
+    default: 0,
+    validator(value: string | number) {
+      const val = toNumber(value)
+      return val >= 0 && val <= 24
+    },
+  },
+  direction: {
+    type: String as PropType<'horizontal' | 'vertical'>,
+    default: 'vertical',
+    validator(value: string) {
```

Comment:
We had removed all validators and simplified types in props.ts.  You can find all changes in recent commits.
![image](https://github.com/varletjs/varlet/assets/70570907/8a161e67-7801-4999-847b-9ea242a24f9c)

Reply:
ok~ I'm done